main2.py

```diff
@@ -0,0 +1,99 @@
+import random
+used_letters = []
+user_word = []
+word_list = []
+lifes = 5
+
+
+def decor(func):

```

Comment:
Złe użycie dekoratora, Jeżeli używasz go normalnie jako w funkcji to nie pisz dekoratora tylko zrób normalnie wywłoanie w funkcj. Fajnie że się tego nauczyłeś ale to nie do tego służy